indico/modules/categories/client/js/components/CategoryStatistics.jsx

```diff
@@ -0,0 +1,196 @@
+    [data]
+  );
+
+  return (
+    <Container>
+      <Segment.Group horizontal>
+        <Segment>
+          <Header as="h2">
+            <Header.Content>
+              <Translate>Number of events</Translate>
+              <Header.Subheader>
+                <Translate>The year is the one of the start date of the event.</Translate>
+              </Header.Subheader>
+            </Header.Content>
+          </Header>
+          <Segment basic>
+            {eventGraph[0].data.length === 0 ? (
+              <Message info>
+                <Translate>No data available</Translate>
+              </Message>
+            ) : (
+              <div styleName="chart-container">
+                <Chart
+                  styleName="chart"
+                  data={eventGraph}
+                  series={series}
+                  axes={axes}
+                  tooltip={tooltip}
+                />
+              </div>
+            )}
+          </Segment>
+        </Segment>
+        <Segment>
+          <Header as="h2">
+            <Header.Content>
+              <Translate>Number of contributions</Translate>
+              <Header.Subheader>
+                <Translate>The year is the one of the start date of the contribution.</Translate>
+              </Header.Subheader>
+            </Header.Content>
+          </Header>
+          <Segment basic>
+            {contributionsGraph[0].data.length === 0 ? (
+              <Message info>
+                <Translate>No data available</Translate>
+              </Message>
+            ) : (
+              <div styleName="chart-container">
+                <Chart
+                  styleName="chart"
+                  data={contributionsGraph}
+                  series={series}
+                  axes={axes}
+                  tooltip={tooltip}
+                />
+              </div>
+            )}
+          </Segment>
+        </Segment>
+      </Segment.Group>
+      <Segment.Group horizontal>
+        <Segment inverted color="blue" textAlign="center">
+          <Statistic inverted>
+            <Statistic.Value>{totalEvents}</Statistic.Value>
+            <Statistic.Label>
+              <Translate>Total number of events</Translate>
+            </Statistic.Label>
+          </Statistic>
+        </Segment>
+        <Segment inverted color="blue" textAlign="center">
+          <Statistic inverted>
+            <Statistic.Value>{totalContributions}</Statistic.Value>
+            <Statistic.Label>
+              <Translate>Total number of contributions</Translate>
+            </Statistic.Label>
+          </Statistic>
+        </Segment>
+        <Segment inverted color="blue" textAlign="center">
+          <Statistic inverted>
+            <Statistic.Value>{files}</Statistic.Value>
+            <Statistic.Label>
+              <Translate>Number of attachments</Translate>
+            </Statistic.Label>
+          </Statistic>
+        </Segment>
+        {data.users && (
```

Comment:
Shouldn't this be also displayed if `data.users == 0` or we only want to show it once `data.users > 0`?

Reply:
you never have 0 users

Reply:
Yeah, I was actually thinking about checking against `null` that is why I added this comment.

Reply:
okay, I will add the null check.

Reply:
After wotking on it, When we go in some subcategory, their number of users is not available and `data.users` is `undefined` and `users` is `NaN`, It satisfies `data.users !== null` and then shows `NAN` like the image below, but when we have just `data.users`, it will not pass the `NAN` value and the segment will not be shown. As @ThiefMaster mentioned, we will never have `0` users, `data.users` will never be `null`, It may be `undefined` in case of sub category.  Do we still `null` check?
![Screenshot from 2020-04-01 12-24-49](https://user-images.githubusercontent.com/24358501/78127090-daaa0e80-7413-11ea-92ed-63ce202d310a.png)

Reply:
then check for `undefined` instead of `null`... :)